build.gradle.kts

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation("org.bstats:bstats-bukkit:3.0.1")
     implementation("com.jeff_media:SpigotUpdateChecker:3.0.3")
 
-    compileOnly("org.spigotmc:spigot-api:1.20.6-R0.1-SNAPSHOT")
+    compileOnly("org.spigotmc:spigot-api:1.21.1-R0.1-SNAPSHOT")
```

Comment:
Is this change necessary? After 1.20.6, Spigot starts to add their relocation stuff for their interface changes. By updating to 1.21.1 here, we are opting out... I don't *think* we are using any of those interfaces, but definitely worth a check.

Reply:
Ah gotcha, I'll check if it's need!

Reply:
Yeah you are correct. It's not required.